src/Pdf.php

```diff
@@ -35,13 +36,22 @@ public function __construct(string $pdfFile)
             throw new PdfDoesNotExist("File `{$pdfFile}` does not exist");
         }
 
-        $this->imagick = new Imagick($pdfFile);
+        $this->imagick = new Imagick();
 
-        $this->numberOfPages = $this->imagick->getNumberImages();
+        $this->numberOfPages = $this->fetchNumberPages($pdfFile);
 
         $this->pdfFile = $pdfFile;
     }
 
+    public function fetchNumberPages(string $pdfFile)
+    {
+        $parser = new Parser();
```

Comment:
What code is being executed in that package? My it's simple enough to transfer to this package, so we don't have take on a dependency.

Reply:
There big class that parse pdf document. Alternatively can use unix tool pdfinfo and then run exec('phpinfo '.$filename);